### PR TITLE
[ecs] Add Atlantis for GitOps

### PR DESCRIPTION
**Pull request by maximmi**
## what
* Add atlantis to root.cloudposse.co

## why
* Support operations by pull request (GitOps)

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,38 +1,47 @@
...
 
 # Terraform vars
-ENV TF_VAR_region="${AWS_REGION}"
-ENV TF_VAR_account_id="${AWS_ACCOUNT_ID}"
-ENV TF_VAR_namespace="cpco"
-ENV TF_VAR_stage="root"
-
-ENV TF_VAR_parent_domain_name="cloudposse.co"
-ENV TF_VAR_root_domain_name="root.cloudposse.co"
+#ENV TF_VAR_region="${AWS_REGION}"
```
Comment (osterman) [inline Dockerfile]: Just delete these

**Comment by osterman** [comment]
atlantis plan

**Comment by osterman** [comment]
atlantis plan

**Comment by osterman** [comment]
atlantis plan

**Comment by osterman** [comment]
atlantis plan

**Comment by goruha** [comment]
atlantis plan

**Pull request merged**